google/cloud/bigtable/CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## v1.9.x - TBD
+## v1.10.x - TBD
 
-## v1.8.x - 2020-02
+## v1.9.x - 2020-4
+
```

Comment:
Please include

```
> **NOTICE:** This repo will soon contain the code for all the other related
`google-cloud-cpp-*` repos. As a new monorepo
([#3612](https://github.com/googleapis/google-cloud-cpp/issues/3612)), the
versioning of this repo will be changing to have a single per-repo version. See
https://github.com/googleapis/google-cloud-cpp/issues/3615 for more info.
```

Reply:
Correction: I think we should tweak the wording a bit. So how about including the following in both `bigtable/CHANGELOG.md` and `storage/CHANGELOG.md`.

```
> **NOTICE:** This repo will soon contain the code for all the other related
`google-cloud-cpp-*` repos. As a new monorepo
([#3612](https://github.com/googleapis/google-cloud-cpp/issues/3612)), the
versioning of this repo will be changing to have a single per-repo version.
**The per-library version numbers will be removed in favor of the repo version.** See
https://github.com/googleapis/google-cloud-cpp/issues/3615 for more info.
```